### PR TITLE
Document MM-48538: `/apps debug logs` to stream plugin logs to a Mattermost channel

### PR DESCRIPTION
**Pull request by neflyte**
## Summary

This PR documents MM-48538 by way of https://github.com/mattermost/mattermost-plugin-apps/pull/394. It is a replacement for #1178.

A new section, "Debug Apps", has been added to capture App debugging commands and related information.

Preview link: http://mattermost-dev-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/1181/integrate/apps/functionality/debug/

### CODE REVIEW

**Comment by levb** [comment]
@neflyte ditto please reconcile with and close https://github.com/mattermost/mattermost-developer-documentation/pull/1178

File: site/content/integrate/apps/functionality/_index.md
Comment (cwarnermm) [inline site/content/integrate/apps/functionality/_index.md:16]:
```suggestion
- [Debug Apps]({{<ref "debug">}}) - learn how to debug your App.
```

**Review by cwarnermm — APPROVED** [review]
Added punctuation feedback. Approving.

File: site/content/integrate/apps/functionality/debug.md
Comment (hanzei) [inline site/content/integrate/apps/functionality/debug.md:11]: That is no longer true. Since https://github.com/mattermost/mattermost-plugin-apps/pull/372 Development mode is enabled via `/apps settings`.

**Review by hanzei — APPROVED** [review]

File: site/content/integrate/apps/functionality/debug.md
Comment (hanzei) [reply site/content/integrate/apps/functionality/debug.md:11]: Not, https://github.com/mattermost/mattermost-plugin-apps/pull/372 is not released yet. It will be in the same release as https://github.com/mattermost/mattermost-plugin-apps/pull/394.

Comment (asatkinson) [reply site/content/integrate/apps/functionality/debug.md:11]: @neflyte this content is fine, but should not be pushed to production until Apps Framework 1.2.  This should be in the next 1-2 weeks.

Comment (asatkinson) [reply site/content/integrate/apps/functionality/debug.md:11]: @hanzei, I've been told this will be released in 1.2 in the next 1-2 weeks.  We should hold this until then, but

Comment (levb) [reply site/content/integrate/apps/functionality/debug.md:11]: 0/5 I don't see why not - let's just get them both merged quickly.

**Comment by azigler** [comment]
Nice addition to the docs, @neflyte! 👍  Looks ready to merge once 1.2 is released.

**Review by azigler — APPROVED** [review]

**Comment by cwarnermm** [comment]
@asatkinson - This PR has a "do not merge/awaiting release" label. What release is this PR waiting on? Is it possible to merge this PR in now?

**Comment by cwarnermm** [comment]
Confirmed with @azigler that this PR is waiting for apps framework 1.2 to be merged and deployed into the Marketplace repo https://github.com/mattermost/mattermost-marketplace/pull/331

File: site/content/integrate/apps/functionality/debug.md
Comment (hanzei) [reply site/content/integrate/apps/functionality/debug.md:11]: Updated the docs in https://github.com/mattermost/mattermost-developer-documentation/pull/1211

**Pull request merged**
